packages/market-service/src/index.ts

```diff
@@ -1 +1,2 @@
 export * from './market-service-manager'
+export * from './exchange-rates-host/exchange-rates-host'
```

Comment:
i do wonder if this should be part of the market service api, much like the crypto services are, so we can have multiple backing sources to avoid rate limiting/downtime. 

what do you think?

Reply:
Actually, in my first approach, I wrote a similar market-service-manager, but only for the fiat-rate providers, it seemed a little bit too much because we only got one provider right now, but I'm ok if you want it to be implemented that way for easier future provider integrations.

Reply:
@stackedq after discussing with @cjthompson lets please go ahead and create this abstraction, in the same way the market service currently does it to source crypto prices, such that we can add redundancy/extra capability transparently in future.

Reply:
done